apps/desktop/bi2p/src/main/resources/css/base.css

```
@@ -0,0 +1,140 @@
+    -fx-faint-focus-color: -bisq2-green;
+    -fx-default-button: -bisq2-green-dim-20;
+    -bisq-green-hover: -bisq2-green-dim-10;
+    -bisq-green-pressed: -bisq2-green-dim-30;
+
+    -fx-dark-text-color: -bisq-black;
+    -fx-mid-text-color: -bisq-mid-grey-20;
+    -fx-light-text-color: -bisq-white;
+
+    -bisq-error: -bisq2-red;
+
+    -fx-text-background-color: ladder(
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)